starters/deno-oak-denodb/README.md

```diff
@@ -37,6 +37,29 @@ This starter kit can be used for scaffolding an all-Deno based backend. It uses
 
 ## Installation
 
+### CLI (Recommended)
+
+```shell
+npm create @this-dot/starter -- --kit deno-oak-denodb
+```
+
+or
+
+```shell
+yarn create @this-dot/starter -- --kit deno-oak-denodb
+```
+
+- Follow the prompts to select the `deno-oak-denodb` starter kit and name your new project.
```

Comment:
passing the flag `-- --kit deno-oak-denodb` should not bring up the prompt to select the kit. It takes you to name your project.

Reply:
Good catch, fixed.